libs/csm/rest/csm_rest_quota.py

```diff
@@ -124,7 +122,7 @@ def verify_get_set_user_quota(self, uid: str, payload: dict, verify_response=Fal
                     if key in ("enabled","max_size","max_objects","check_on_raw"):
                         continue
                     if key == "max_size_kb":
-                        payload.update({"max_size_kb":get_response["max_size"]/1000})
+                        payload.update({"max_size_kb":round(get_response["max_size"]/1024)})
```

Comment:
Can u once check this behaviour with RGW?
I think for RGW it is 1024

Reply:
yes, I have used 1024 only